cmd/launcher/internal/updater/updater.go

```diff
@@ -62,12 +63,29 @@ func NewUpdater(
 		return nil, err
 	}
 
+	// create the new updater
+	tufAutoupdater, err := autoupdate.NewTufAutoupdater(
```

Comment:
I'm fine with a new pkg as well! I started out here because I wasn't sure how much of the existing autoupdate functionality I wanted to lean on, but even with the full download/stage I only used `checkExecutable`.

Would definitely like suggestions for naming, `tufautoupdate` versus `autoupdate` is feeling a little murky

Reply:
I went with package `tuf` and have started aliasing the other tuf to `legacytuf`, but I am not 100% sold on it and open to suggestions!

Reply:
I would be unclever and name it `autoupdate2`. Or possible `autoupdate/tuf`. Or possible it doesn't matter, since we could, eventually, rename it.

Reply:
Moved under autoupdate/tuf